cmake-modules/AddAzureSDKforC.cmake

```diff
@@ -17,7 +17,7 @@
 include(FetchContent)
 FetchContent_Declare(azuresdkforc
     GIT_REPOSITORY      https://github.com/Azure/azure-sdk-for-c.git
-    GIT_TAG             1.0.0)
+    GIT_TAG             1.1.0-beta.1)
```

Comment:
Since we haven't release this tag yet, please revert this for now.

Reply:
That is the intent, so that when we release it, it is not broken. Because, for the actual 1.0.0. release, we point to 1.0.0-prewiew.6, which does not exist and never existed as well.

Reply:
It is broken for the repo version anyway - because what we have in the repo is not 1.0.0 release.
With this change, when we release beta, it won't be broken, as it was with the release:
https://github.com/Azure/azure-sdk-for-c/commits/256bc6fdc4fcbe7a7b31a1b745ad5b9be6c54a0b/cmake-modules/AddAzureSDKforC.cmake

Reply:
As long as we update that version during the next release, I'm fine. Although, for the random version you get from github, it also does not point to the exact commit it was pointing at the time of your pull